FT-Telemetry metrics change to reflect spec update

Azquelt opened the pull request:
Implements the spec changes proposed in eclipse/microprofile-fault-tolerance#640.

Will fail the TCK until updated tests are added.

- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Azquelt committed:
FT-Telemetry metrics change to reflect spec update

benjamin-confino commented:
I see that every change you makes matches a change in https://github.com/eclipse/microprofile-fault-tolerance/pull/640/files

And every change in https://github.com/eclipse/microprofile-fault-tolerance/pull/640/files has a change here.

Looking good.

Azquelt committed:
Disable tests until new FT TCK is available

We're making implementation changes to match changes in the spec, but
that means our implementation won't pass the old version of the TCK.

Azquelt commented:
#build

Azquelt commented:
Errors in the build are known failures.
